js/node/test/unittests/lib/inference-session.ts

```diff
@@ -68,6 +68,11 @@ describe('UnitTests - InferenceSession.create()', () => {
       await InferenceSession.create(new Uint8Array(0));
     }, {name: 'Error', message: /No graph was found in the protobuf/});
   });
+  it('EXPECTED FAILURE - invalid buffer', async () => {
+    await assert.rejects(async () => {
+      await InferenceSession.create(new Uint8Array(Array.from({length: 100}, () => 42)));
+    }, {name: 'Error', message: /as ONNX.+as ORT/m});
+  });
```

Comment:
Did you actually run this test? It may not work as this part is the unittest for ORT node.js binding but not ORT web.

Currently the best place for ORT-web model loading test expected failure is js\web\test\e2e\browser-test-webgl.js

Reply:
Oops, fixed.